OpenRA.Mods.Common/Traits/Attack/AttackTurreted.cs

```diff
@@ -20,12 +20,14 @@ public class AttackTurretedInfo : AttackFollowInfo, Requires<TurretedInfo>
 		public override object Create(ActorInitializer init) { return new AttackTurreted(init.Self, this); }
 	}
 
-	public class AttackTurreted : AttackFollow
+	public class AttackTurreted : AttackFollow, INotifyCreated
 	{
 		protected Turreted[] turrets;
 
 		public AttackTurreted(Actor self, AttackTurretedInfo info)
-			: base(self, info)
+			: base(self, info) { }
+
+		public void Created(Actor self)
```

Comment:
As I explained it over IRC - it shouldn't be but it does. `Turreted` traits defined after `AttackTurreted` (if there's a `Turreted` trait before the `AttackTurreted` as well) will don't get their facing updated later when disabled.